kubernetes/AKS/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: model-store-claim
+spec:
+  storageClassName: default
```

Comment:
Can you please help me with some info on how storage classes work in AKS.  For EKS i had to install a driver and make the storageClass to a shared volume. How does default map to a shared volume.

Did find some similar ideas in - https://docs.microsoft.com/en-us/azure/aks/concepts-storage (AzureFiles). Wondering how these would be used.